scripts/validate.py

```diff
@@ -16,6 +19,62 @@
         validate.py ./cfr.json ./schema/cfr.schema.json
 """
 
+def intersect_op(listA, listB):
```

Comment:
This also could be defined as a lambda.

`jexl.add_binary_operator('intersect', 40, lambda x, y: set(x).intersection(y))`